### ISSUE TITLE
Lastest build / auto build ?

### ISSUE THREAD

**Issue by GunGunGun** [OtterBrowser/otter-browser#1743]
I think we may want to setup Github action to autobuild new version because the current newest build was from Feb 2022 https://github.com/OtterBrowser/otter-browser/releases

Or if it's possible I hope you'll update release section.

**Comment by Emdek** [comment]
@GunGunGun, yeah, I've considered that already, but it requires some time to pull off…
So lets repurpose this ticket as a task with a "help wanted" tag, it should be fairly easy, and combined with [deploy.py](https://github.com/OtterBrowser/otter-browser/blob/master/packaging/deploy.py) we can get proper packages / installers too.

**Comment by ericjs** [comment]
So is building from the latest source rather than using the last release what is recommended for users? There is remarkably little guidance for users here wrt the status of the project. I see commits which suggests the project isn't dead, but the last release is 2 years old, the readme has a link to "the official forum" which is dead (domain for sale page), etc.

**Comment by ericjs** [comment]
Ok, well I run linux and can do such a build. But without explicit advice about this in the readme, one assumes that the last release is still the most stable choice, and builds from newer source are going to be a crapshoot as to how broken they are. If build-your-own is officially what you are expected to do, and we can expect such builds to be reasonably stable, this should be explicitly stated in the readme.

**Comment by Crystal-RainSlide** [comment]
Provide build guide for popular Linux distros, and maybe macOS & Windows can also be very helpful.
